lib/travis/build.rb

```diff
@@ -49,6 +49,8 @@ def by_lang(lang)
           Script::ObjectiveC
         when 'bash', 'sh', 'shell', 'minimal' then
           Script::Generic
+        when 'elm' then
```

Comment:
I don't think this `when` clause is necessary, as `elm` should translate to `Script::Elm` naturally.